packages/assets-controllers/src/NftController.ts

```diff
@@ -222,41 +218,235 @@ const controllerName = 'NftController';
 /**
  * The external actions available to the {@link NftController}.
  */
-type AllowedActions =
+export type AllowedActions =
```

Comment:
We don't want this to be a package-level export, since it would conflict with the `AllowedActions` we have in other controllers. 

We can leave the `export` keyword here, and fix the asterik export in `index.ts`, making sure the enumerated list of exports excludes `AllowedActions`.

@mcmire Maybe we could consider making `import type { AllowedActions as SomeControllerAllowedActions } ` a norm like `import from { defaultState as defaultNetworkState }` if we want the allowlists to be available externally? Is this something we've been avoiding for security reasons?

Reply:
No, I just think we've never had the need to use `AllowedActions` on its own, that's why we've never exported it. We usually need all of the actions, not just `AllowedActions`. So my opinion is that it's more valuable to export `<ControllerName>Actions` and `<ControllerName>Events`, but I could be convinced.

Reply:
Makes sense. Sounds like that's a bridge we can cross if we get to it. Exporting with the controller name would be my first choice as well if we expect them to be used externally.